### PR TITLE
FunC v0.5.0

### PR DESCRIPTION
**Pull request by tolk-vm**
Add "0.5.0" to `config.json` for both frontend and backend to know it [exists](https://github.com/ton-blockchain/ton/pull/1026).

> **When https://github.com/ton-community/func-js-bin/pull/1 and https://github.com/ton-community/contract-verifier-sdk/pull/324 are merged, this MR will become non-draft**

### CODE REVIEW

**Review by Yoga077 — APPROVED** [review]

**Review by Yoga077 — APPROVED** [review]

File: config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "backends": ["https://ton-source-prod-1.herokuapp.com","https://ton-source-prod-2.herokuapp.com","https://ton-source-prod-3.herokuapp.com"],
   "backendsTestnet": ["https://ton-source-prod-testnet-1.herokuapp.com"],
-  "funcVersions": ["0.4.4", "0.4.3", "0.4.2", "0.4.1", "0.4.0", "0.3.0", "0.2.0"],
+  "funcVersions": ["0.5.0", "0.4.4", "0.4.3", "0.4.2", "0.4.1", "0.4.0", "0.3.0", "0.2.0"],
```
Comment (Yoga077) [inline config.json:4]: Done

**Review by Yoga077 — APPROVED** [review]

**Review by SupernLucky — APPROVED** [review]

**Review by SupernLucky — APPROVED** [review]

**Review by SupernLucky — APPROVED** [review]

**Review by Agulyas1994 — APPROVED** [review]

**Review by Agulyas1994 — APPROVED** [review]

**Review by llllv — APPROVED** [review]

**Review by llllv — APPROVED** [review]

**Review by mm8191 — APPROVED** [review]

File: config.json (hunk 1 above)
Comment (Agulyas1994) [reply config.json:4]: Agulyas

**Review by Agulyas1994 — APPROVED** [review]

File: config.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "backends": ["https://ton-source-prod-1.herokuapp.com","https://ton-source-prod-2.herokuapp.com","https://ton-source-prod-3.herokuapp.com"],
   "backendsTestnet": ["https://ton-source-prod-testnet-1.herokuapp.com"],
-  "funcVersions": ["0.4.4", "0.4.3", "0.4.2", "0.4.1", "0.4.0", "0.3.0", "0.2.0"],
```
Comment (Agulyas1994) [inline config.json:4]: 0.50

**Review by Agulyas1994 — APPROVED** [review]

**Review by Agulyas1994 — APPROVED** [review]

**Review by tiaraton — APPROVED** [review]

**Review by Meroksa — APPROVED** [review]

**Review by Meroksa — APPROVED** [review]

**Review by Meroksa — APPROVED** [review]

**Review by Meroksa — APPROVED** [review]

**Review by Elgayar777 — APPROVED** [review]
- [ ]

File: config.json (hunk 1 above)
Comment (Elgayar777) [reply config.json:4]: - [ ]

File: config.json (hunk 2 above)
Comment (Elgayar777) [reply config.json:4]: Done

**Review by MrG9090 — APPROVED** [review]

**Pull request closed without merging**